core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1004,6 +1004,7 @@ private[spark] object Utils extends Logging {
           for (child <- listFilesSafely(file)) {
             try {
               deleteRecursively(child)
+              logTrace(file.getAbsolutePath + " has been deleted")
```

Comment:
I mean, why not `s"${file.getAbsolutePath} has been deleted"`? it's actually important for a log trace statement